examples/blob_binding.cpp

```diff
@@ -1,4 +1,6 @@
+#include <sqlite3.h>
```

Comment:
When consuming sqlite_orm as a module, it doesn't, but only symbols that are exported from the module. Which is good, because it doesn't pollute the (global) namespace anymore.

Reply:
got it. Thanks